hack/run-latency-testing.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-GINKGO_SUITS=${GINKGO_SUITS:-functests/5_latency_testing}
+GINKGO_SUITS=${GINKGO_SUITS:-"./test/e2e/performanceprofile/functests/0_config ./test/e2e/performanceprofile/functests/5_latency_testing"}
```

Comment:
why do we need the config suite?

Reply:
to create the performance profile, other wise tests will be skipped

Reply:
uhm, makes sense because here we're testing the latency test, but I can't recall why we didn't add the config before. Let's check that.

Reply:
before this change there was no make target.
the original one I proposed is just like the PAO one:
https://github.com/openshift-kni/performance-addon-operators/blob/master/Makefile#L267
but I was suggested to organize better the target like this and put the suites to execute inside the script:
https://github.com/openshift/cluster-node-tuning-operator/pull/376#discussion_r899975859